src/fileseq/frameset.py

```diff
@@ -4,6 +4,7 @@
 """
 from __future__ import absolute_import
 
+from builtins import range
```

Comment:
unnecessary import?

Reply:
Ooops! Good catch!

My original implementation used `range()`. I will remove this.